client/java/src/main/java/io/openlineage/client/utils/DatasetNamespaceCombinedResolver.java

```diff
@@ -0,0 +1,91 @@
+/*
+/* Copyright 2018-2024 contributors to the OpenLineage project
+/* SPDX-License-Identifier: Apache-2.0
+*/
+
+package io.openlineage.client.utils;
```

Comment:
Why not have this class in the package together with rest of the relevant classes?